tests/test_data/valid_case_insensitive/pixels/pixel_subfolder/test_pixels.json

```diff
@@ -0,0 +1,64 @@
+{
+    // Collection of pixels from various platforms for testing
+    // Note: definitions may not be fully accurate
+    "m.windows.crash.stable": {
```

Comment:
Just to keep it simple, we can probably keep a simplified version of the `crash` pixel only and also keep this file outside the sub-folder (we test nested folders elsewhere)